src/main/webapp/event-info-page/event-info.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
This file has already been added it appears, just a slightly different name. If you are attempting to rename the file from eventinfo.html to event-info.html, can you delete the file eventinfo.html??? Or if this was added by accident, can you delete this file?

Reply:
Thanks for point out! I will delete eventinfo.html